release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -103,6 +103,11 @@ Warning: short name "ex" could also match lower priority resource examples.test.
 [id="ocp-4-16-networking"]
 === Networking
 
+[id="ocp-4-16-networking-connectivity-pod-placement"]
+==== Configurable placement of network connectivity check pods
+
+To periodically test network connectivity amongst cluster components, the Cluster Network Operator (CNO) creates `network-check-source` deployment and the `network-check-target` daemon set. In {product-title} 4.16, you can configure the nodes where both the source and target pods for connectivity checks are run by setting node selectors. For more information, see xref:../networking/verifying-connectivity-endpoint.adoc#verifying-connectivity-endpoint[Verifying connectivity to an endpoint].
```

Comment:
```suggestion
To periodically test network connectivity amongst cluster components, the Cluster Network Operator (CNO) creates the `network-check-source` deployment and the `network-check-target` daemon set. In {product-title} 4.16, you can configure the nodes by setting node selectors and run the source and target pods to check the network connectivity. For more information, see xref:../networking/verifying-connectivity-endpoint.adoc#verifying-connectivity-endpoint[Verifying connectivity to an endpoint].
```

Reply:
A passive voice can be avoided here -- assuming that a user is the one who sets node selectors for nodes.